### PR TITLE
add miq-ansibleapp

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jmontleon** [comment]
Currently launches and runs.

Login page is displayed, but when I try to log in I see in the production.log:
`[----] W, [2017-02-23T20:05:56.868527 #1796:2b1ee5e458d4]  WARN -- : Can't verify CSRF token authenticity.
[----] F, [2017-02-23T20:05:56.868864 #1796:2b1ee5e458d4] FATAL -- : Error caught: [ActionController::InvalidAuthenticityToken] ActionController::InvalidAuthenticityToken
`
I am reaching out to Franco Bladilo to see if he may have some insight as to what is going wrong.

File: miq-ansibleapp/ansibleapp.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    type: string
+    default: 1M
+  - name: memcached_service_name 
+    description: Service name of the memcached instance to use
+    type: string
+    default: miq-memcached
+  - name: miq_memcached_port
+    description: Memcached port
+    type: string
+    default: "tcp://{{ memcached_service_name }}:11211"
```
Comment (eriknelson) [inline miq-ansibleapp/ansibleapp.yml]: Will this actually work? I'm not sure, but it's a sensible requirement.

Comment (jmontleon) [reply miq-ansibleapp/ansibleapp.yml]: Oh, it won't. I had to fix it in the defaults/main.yml. I'll fix it here as well. Thanks for catching this.

**Comment by jmontleon** [comment]
@eriknelson I removed these from ansibleapp.yml and did some additional changes to the default.yml.

It looks like things are passing through correctly on the pod.

**Comment by eriknelson** [comment]
ACK

**Pull request merged**